ciao-launcher/main.go

```diff
@@ -97,12 +99,8 @@ func init() {
 	flag.StringVar(&serverURL, "server", "", "URL of SSNTP server")
 	flag.StringVar(&serverCertPath, "cacert", "/etc/pki/ciao/CAcert-server-localhost.pem", "Client certificate")
 	flag.StringVar(&clientCertPath, "cert", "/etc/pki/ciao/cert-client-localhost.pem", "CA certificate")
-	flag.StringVar(&computeNet, "compute-net", "", "Compute Subnet")
-	flag.StringVar(&mgmtNet, "mgmt-net", "", "Management Subnet")
 	flag.Var(&networking, "network", "Can be none, cn (compute node) or nn (network node)")
 	flag.BoolVar(&hardReset, "hard-reset", false, "Kill and delete all instances, reset networking and exit")
-	flag.BoolVar(&diskLimit, "disk-limit", true, "Use disk usage limits")
-	flag.BoolVar(&memLimit, "mem-limit", true, "Use memory usage limits")
```

Comment:
Yep I think they should be true by default.  Otherwise launcher will not generate snntp.FULL messages and a node could become overwhelmed.  Even if we were to switch the defaults to false, we should do this in a separate commit so it's obvious that a change has been made.